components/product-list-item-details.tpl

```diff
@@ -10,11 +10,33 @@
       {%- if _buyButton.product.uses_variants == true -%}
         {{ look_closer_btn }}
         <div class="product-item-price">
-          {%- if _buyButton.product.price_max_with_tax != _buyButton.product.price_min_with_tax -%}
-            {{ _buyButton.product.price_min_with_tax | money_with_currency: _buyButton.product.currency }}
-            <span>—</span>
-          {%- endif -%}
-          {{ _buyButton.product.price_max_with_tax | money_with_currency: _buyButton.product.currency }}
+          {%- capture original_price -%}
+            {%- if _buyButton.product.price_max_with_tax != _buyButton.product.price_min_with_tax -%}
+              {{ _buyButton.product.price_min_with_tax | money_with_currency: _buyButton.product.currency }}
+              <span> – </span>
+            {%- endif -%}
+            {{ _buyButton.product.price_max_with_tax | money_with_currency: _buyButton.product.currency }}
+          {%- endcapture -%}
+
+          <div class="product-price">
```

Comment:
This div (`<div class="product-price">`) adds additional padding and should be removed.

<img width="691" alt="Screenshot 2023-10-24 at 15 50 47" src="https://github.com/Voog/design-anchorage/assets/378755/8a1a019f-5a50-41c5-b97c-82d80d084dfd">

```suggestion
```